test/ContentHubClientTest.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+namespace Acquia\ContentHubClient\test;
+
+
+use Acquia\ContentHubClient\ContentHubClient;
+use GuzzleHttp\Exception\GuzzleException;
+use GuzzleHttp\Psr7\Response;
+use PHPUnit\Framework\TestCase;
+
+class ContentHubClientTest extends TestCase
+{
+  /**
+   * @var ContentHubClient
+   */
+  private $contentHubClient;
+
+  /**
+   *
+   */
+  public function setUp() : void
+  {
+    parent::setUp();
+
+    $this->contentHubClient = \Mockery::mock(ContentHubClient::class)
+      ->makePartial();
+  }
+
+  /**
+   *
+   */
+  public function tearDown() : void
+  {
+    parent::tearDown();
+    unset($this->contentHubClient);
+    \Mockery::close();
+    gc_collect_cycles();
+  }
+
+  public function testDefinition()
+  {
+    $successJsonResponse = \GuzzleHttp\json_encode([
+      'response_code' => 200,
```

Comment:
Use constants instead of pure http response codes.

Reply:
done

Reply:
These constants are already defined for you in Response class (The Symfony one not that of Guzzle)

Reply:
fixed